content/it/docs/reference/glossary/job.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Job
+id: job
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/jobs-run-to-completion
+short_description: >
+  Un'attività finita ovvero un insieme di queste che vengono eseguite fino al completamento.
```

Comment:
I dont' like this sentence, but I don't have better suggestion ATM...

Reply:
What about this? "Uno o più lavori(task) che vengono eseguiti fino al loro completamento."